frontend/components/Teams/SeparateTeam.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default class SeparateTeam extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { team: {} };
+  }
+
+  componentDidMount() {
```

Comment:
You can use here hook useRequest from library ahook here https://ahooks.js.org/hooks/async
or hook useEffect from react https://www.robinwieruch.de/react-hooks-fetch-data